### DISCUSSION TITLE
Can't access setup wizard after 15 minutes on DO

### DISCUSSION THREAD

**Discussion by mattgilbertnet** [requarks/wiki#3457] [Help / Questions]
**Question**
I've gone through these [installation instructions](https://docs.requarks.io/install/ubuntu) on a new Digital Ocean Ubuntu droplet with no errors. When I get to the "Access the setup wizard" section, and try to access the setup wizard via the public IP, the tab closes immediately. I've waited over 15 minutes to try again, and I get the same result. The same thing happened when I tried the 1-click Marketplace app (Which is why I tried the manual approach.)

**Host Info (please complete the following information):**
OS: Ubuntu 20.04 (LTS) x64
Wiki.js version: ~~2.4.107~~ EDIT: 2.5.159
Database engine: PostgreSQL 11

**Comment by NGPixel** [comment]
Have you tried to connect to your droplet and run `docker logs wiki`?

**Comment by mattgilbertnet** [comment]
Thanks, @NGPixel ! I'm still stuck, but there were some helpful messages in the log:

> 
> 2020-10-10T11:01:26.012Z [MASTER] info: =======================================
> 2020-10-10T11:01:26.014Z [MASTER] info: = Wiki.js 2.5.159 =====================
> 2020-10-10T11:01:26.015Z [MASTER] info: =======================================
> 2020-10-10T11:01:26.015Z [MASTER] info: Initializing...
> 2020-10-10T11:01:27.195Z [MASTER] info: Using database driver pg for postgres [ OK ]
> 2020-10-10T11:01:27.199Z [MASTER] info: Connecting to database...
> 2020-10-10T11:01:27.231Z [MASTER] info: Database Connection Successful [ OK ]
> 2020-10-10T11:01:27.781Z [MASTER] warn: DB Configuration is empty or incomplete. Switching to Setup mode...
> 2020-10-10T11:01:27.782Z [MASTER] info: Starting setup wizard...
> 2020-10-10T11:01:28.462Z [MASTER] info: Starting HTTP server on port 3000...
> 2020-10-10T11:01:28.462Z [MASTER] info: HTTP Server on port: [ 3000 ]
> 2020-10-10T11:01:28.466Z [MASTER] info: HTTP Server: [ RUNNING ]
> 2020-10-10T11:01:28.466Z [MASTER] info: 🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻
> 2020-10-10T11:01:28.466Z [MASTER] info: 
> 2020-10-10T11:01:28.467Z [MASTER] info: Browse to http://YOUR-SERVER-IP:3000/ to complete setup!
> 2020-10-10T11:01:28.467Z [MASTER] info: 
> 2020-10-10T11:01:28.467Z [MASTER] info: 🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺
> 2020-10-10T13:27:33.883Z [MASTER] error: Not Found
> TypeError: Cannot read property 'sendError' of undefined
>     at /wiki/server/setup.js:393:20
>     at Layer.handle_error (/wiki/node_modules/express/lib/router/layer.js:71:5)
>     at trim_prefix (/wiki/node_modules/express/lib/router/index.js:315:13)
>     at /wiki/node_modules/express/lib/router/index.js:284:7
>     at Function.process_params (/wiki/node_modules/express/lib/router/index.js:335:12)
>     at next (/wiki/node_modules/express/lib/router/index.js:275:10)
>     at /wiki/server/setup.js:383:5
>     at Layer.handle [as handle_request] (/wiki/node_modules/express/lib/router/layer.js:95:5)
>     at trim_prefix (/wiki/node_modules/express/lib/router/index.js:317:13)
>     at /wiki/node_modules/express/lib/router/index.js:284:7
>     at Function.process_params (/wiki/node_modules/express/lib/router/index.js:335:12)
>     at next (/wiki/node_modules/express/lib/router/index.js:275:10)
>     at /wiki/node_modules/body-parser/lib/read.js:130:5
>     at invokeCallback (/wiki/node_modules/raw-body/index.js:224:16)
>     at done (/wiki/node_modules/raw-body/index.js:213:7)
>     at IncomingMessage.onEnd (/wiki/node_modules/raw-body/index.js:273:7)

And a bunch more of the "not Found" and "sendError" message.

The documentation says to navigate to http://YOUR-SERVER-IP, but the error says http://YOUR-SERVER-IP:3000. When I try http://YOUR-SERVER-IP:3000, the page never loads and I get "This site can’t be reached... took too long to respond." The same "Not Found" and "sendError" messages repeat in the log.

**Comment by NGPixel** [comment]
The DO droplet is already configured to map port 3000 to 80, so you should use http://DROPLET-IP, not :3000.

Can you try to create a new droplet using the official marketplace image? Then leave it 10-15 minutes before attempting to access it via the droplet IP?
